### ISSUE TITLE
Variable sized table rows positioned incorrectly

### ISSUE THREAD

**Issue by PawelGIX** [clauderic/dnd-kit#1118]
When sorting a table with different sized rows, they are not positioned correctly.
It seems that their dimension is not read correctly.

Example:
https://codesandbox.io/s/dnd-kit-vertical-variable-height-removable-draggable-forked-q3i1f8?file=/src/SortableItem.js

**Comment by CodeSmith32** [comment]
Yes, it would appear that settings `strategy` to `verticalListSortingStrategy` on the `<SortableContext>` component fixes this glitch:

```tsx
<SortableContext strategy={verticalListSortingStrategy}>
   {/* ... */}
</SortableContext>
```

For those using variable height draggables, also be sure to use `CSS.Translate.toString` and NOT `CSS.Transform.toString` when styling the sortable elements ([see this issue](https://github.com/clauderic/dnd-kit/issues/117)). Otherwise the elements stretch obscenely.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
